specification/storage/data-plane/Microsoft.FileStorage/stable/2024-11-04/file.json

```diff
@@ -7190,6 +7236,35 @@
       },
       "x-ms-parameter-location": "method"
     },
+    "PaidBurstingEnabled": {
+      "name": "x-ms-share-paid-bursting-enabled",
+      "x-ms-client-name": "paidBurstingEnabled",
+      "description": "Optional. Boolean. Default if not specified is false. This property enables paid bursting.",
+      "in": "header",
+      "x-ms-parameter-location": "method",
+      "required": false,
+      "type": "boolean"
+    },
+    "PaidBurstingMaxBandwidthMips": {
```

Comment:
Same typo here and in refs above (this one probably matters less because it doesn't affect generated code)